libraries/SPISlave/src/hspi_slave.c

```diff
@@ -114,7 +114,7 @@ void hspi_slave_end()
   SPI1P = B110;
 }
 
-void hspi_slave_setStatus(uint32_t status)
+void ICACHE_RAM_ATTR hspi_slave_setStatus(uint32_t status) // GuaAck. 30.10.2018
```

Comment:
You provided an explanation for why this function should have the decorator. I suggest replacing the comment with that.
Also, I suggest inline.

Reply:
Same here, olease change the comment to add the explanation.